src/Timeline.tsx

```diff
@@ -96,274 +81,141 @@ export const Timeline = <EID extends string, LID extends string, E extends Timel
-                            timeScale={timeScale}
-                            yScale={yScale}
-                            expanded={laneDisplayMode === 'expanded'}
-                          />
-                          {laneDisplayMode === 'expanded' ? (
-                            <ExpandedMarks
-                              mouseCursor={mouseCursor}
-                              events={eventsInsideDomain}
-                              lanes={lanes}
-                              timeScale={timeScale}
-                              yScale={yScale}
-                              height={height}
-                              eventComponent={eventComponent}
-                              onEventHover={onEventHoverDecorated}
-                              onEventUnhover={onEventUnhoverDecorated}
-                              onEventClick={onEventClick}
-                              tooltipArrow={tooltipArrow}
-                            />
-                          ) : (
-                            <CollapsedMarks
-                              mouseCursor={mouseCursor}
-                              events={eventsInsideDomain}
-                              timeScale={timeScale}
-                              height={height}
-                              eventComponent={eventComponent}
-                              onEventHover={onEventHoverDecorated}
-                              onEventUnhover={onEventUnhoverDecorated}
-                              onEventClick={onEventClick}
-                              tooltipArrow={tooltipArrow}
-                            />
-                          )}
-                        </>
-                      )}
-                      {trimRange && (
-                        <TrimRange
-                          startX={timeScale(trimRange[0])!}
-                          endX={timeScale(trimRange[1])!}
-                          height={height}
-                          width={width}
-                        />
-                      )}
-                      {interactionMode.type === 'trim' && timeScale && (
-                        <Trimmer
-                          startX={trimRange ? trimRange[0] : maxDomain[0]}
-                          endX={trimRange ? trimRange[1] : maxDomain[1]}
-                          height={height}
-                          width={width}
-                          timeScale={timeScale}
-                          setTrimMode={setTrimHoverMode}
-                          dateFormat={dateFormat}
-                          highlightActiveArea={interactionMode.variant === 'trim pan end'}
-                        />
-                      )}
-                    </g>
-                  )
-                }}
-              </InteractionHandling>
-            )
-          }}
-        </MouseAwareSvg>
+        <svg viewBox={`0 0 ${width} ${height}`} width={width} height={height} style={{ outlineStyle: 'none'}}>
```

Comment:
I am not sure this is the right place to put this - but when I don't put it then we get the blue box selection problem